### PR TITLE
[cs] order properties by complexity

### PR DESCRIPTION
**Pull request by TomasVotruba**
Ref

- https://github.com/Symplify/Symplify/pull/1171
- https://github.com/Symplify/Symplify/pull/1173

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -113,5 +113,7 @@
     "bin": ["bin/rector"],
     "config": {
         "sort-packages": true
-    }
+    },
+    "minimum-stability": "dev",
```
Comment (carusogabriel) [inline composer.json]: We shouldn't do this. As we discussed previously, open the `minimum-stability` is a BC.

Comment (TomasVotruba) [reply composer.json]: These won't get to tagged version

File: packages/PHPUnit/src/Rector/SpecificMethod/AssertCompareToSpecificMethodRector.php [hunk 2]
```diff
@@ -13,6 +13,11 @@
 
 final class AssertCompareToSpecificMethodRector extends AbstractPHPUnitRector
 {
+    /**
+     * @var string
+     */
+    private $activeFuncCallName;
```
[the remark this answers is not in the document]
Comment (TomasVotruba) [reply packages/PHPUnit/src/Rector/SpecificMethod/AssertCompareToSpecificMethodRector.php]: Will be resolved in next PR

**Comment by carusogabriel** [comment]
I really don't know what is the point of ignoring my review :)

**Comment by TomasVotruba** [comment]
I replied

image

**Pull request merged**